### DISCUSSION TITLE
Get Flathub verification

### DISCUSSION THREAD

**Discussion by abuhuraira0099** [Vysp3r/ProtonPlus#88] [General]
Not really  an important thing to do as the feature is beta right now. But why not?

**Comment by Vysp3r** [comment]
I'll check to see how to get verified today. If it's not too complicated, I'll do it today.

**Comment by Vysp3r** [comment]
They could've made it like everyone else and made a DNS verification instead of having to add a TXT file to your website. It's annoying.

**Comment by Vysp3r** [comment]
![image](https://user-images.githubusercontent.com/62019735/224714432-f05d483f-beae-4f19-8a76-6b0f8bd105c2.png)
There you go!
